src/jdom/bluetooth.ts

```diff
@@ -71,7 +74,8 @@ class BluetoothTransport extends JDTransport {
             this._device = devices?.[0]
         } else {
             const device = await bleRequestDevice({
-                filters: [{ services: [BLUETOOTH_JACDAC_SERVICE] }],
+                filters: [{namePrefix:"BBC micro:bit"}],
```

Comment:
filters should be an optional constructor argument to eventually support non-micro:bit BLE clients. will fix later.

Reply:
TODO